lib/python/picongpu/plugins/plot_mpl/phase_space_visualizer.py

```diff
@@ -89,15 +90,19 @@ def visualize(self, ax=None, **kwargs):
         _, meta = self.data
         # prevent multiple rendering of colorbar
         if not self.plt_obj.colorbar:
-            cbar = plt.colorbar(self.plt_obj, ax=ax)
-            cbar.set_label(
+            self.cbar = plt.colorbar(self.plt_obj, ax=ax)
+            self.cbar.set_label(
                 r'$Q / \mathrm{d}r \mathrm{d}p$ [$\mathrm{C s kg^{-1} m^{-2}}'
                 '$')
 
         ax.set_xlabel(r'${0}$ [${1}$]'.format(meta.r, "\mathrm{\mu m}"))
         ax.set_ylabel(
             r'$p_{0}$ [$\beta\gamma$]'.format(meta.p))
 
+    def clear_cbar(self):
+        """Clear colorbar if present."""
+        if self.cbar is not None:
+            self.cbar.remove()
```

Comment:
~~Please add a second empty line in order to keep the code pep8 conform. (That is the error causes by travis-ci.)~~

Reply:
This should be a method and not a independent function.
Please indent the code.

Reply:
*discussed offline:*
Issue was the missing space at the end of the class definition.